src/bindings/js/node/src/addon.cpp

```diff
@@ -10,12 +12,15 @@
 #include "model_wrap.hpp"
 #include "node_output.hpp"
 #include "openvino/openvino.hpp"
-#include "tensor.hpp"
 #include "partial_shape_wrap.hpp"
 #include "preprocess/preprocess.hpp"
+#include "tensor.hpp"
 
 /** @brief Initialize native add-on */
 Napi::Object InitAll(Napi::Env env, Napi::Object exports) {
+    auto data = new AddonData();
```

Comment:
The lifetime of data is managed by `Napi::Env` ?

Reply:
`Napi::Env` has a default finalizer method that will simply use `delete`  to destroy `AddonData`  when node-addon is unloaded.